### ISSUE TITLE
latex-to-unicode tab completion, round 2; maybe there should be some options?

### ISSUE THREAD

**Issue by ExpandingMan** [JuliaEditorSupport/julia-vim#245]
Kind of hate to open this issue, because I greatly appreciate Carlo's incredible effort at fixing it (and yes, it is really fixed for the previous case), but I think we need some kind of set of options here.

I've been experimenting with dropping deoplete for [completion-nvim](https://github.com/nvim-lua/completion-nvim) (because deoplete is a big pain in the ass with LSP) only to find that exactly the same problem I was having before, in which latex-to-unicode tab completion doesn't work in the presence of completion suggestions, occurs with completion-nvim, even though it is now fixed for deoplete.

Since we've already been down a rabbit hole here, I don't think we should go crazy trying to get one configuration that just magically works for everything.  Rather, would it be possible to introduce a set of options for which the status quo is the default?  I'm not entirely sure what those options should be, but I know Carlo attempted a few different things before my previous issue is fixed, so I'm wondering if some of these should be options.  Otherwise, I have a feeling it's only a matter of time before somebody else has a problem with this that can't be fixed without code changes.

**Comment by ExpandingMan** [comment]
Came back around to messing with this today... is there some reason why this is always done with `<tab>` rather than making it possible to bind it to an arbitrary key combination?

**Comment by ExpandingMan** [comment]
I have figured out the "brute force" work around
```
inoremap <silent><expr> <tab> pumvisible() ? "\<C-n>\<C-x>\<C-u>" : "\<tab>"
```
All of the most crucial functionality (latex-to-unicode as you type, latex-to-unicode with tab, auto-completion with any package with tab) works with this, but it does have some slightly weird side effects in that the selection in the completion popup box is no longer consistent (presumably because it gets cancelled by the other keys).

What I think would be a "permanent, comprehensive fix that works for everything" would be able to bind `<C-n>` to the completion functionality *AND* latex to unicode completion.  This should basically do what the above does, but without the extra key bindings that break the popup selection.  To do this, it would need to be possible to rebind the latex completion.

I have also tried to just bind `<C-n>` to `<C-x><C-u` but this does not work.  I don't know why.
